safe_control_gym/controllers/mpc/mpc.py

```diff
@@ -407,14 +489,14 @@ def run(self,
             self.results_dict['action'].append(action)
             self.results_dict['state'].append(env.state)
             self.results_dict['state_mse'].append(info['mse'])
-            # self.results_dict['state_error'].append(env.state - env.X_GOAL[i,:])
-
+            if self.solver == 'ipopt':
+                self.results_dict['state_error'].append(env.state - env.X_GOAL[i, :])
             common_metric += info['mse']
```

Comment:
common_metric?? I know this wasnt added in this PR but this seems like a super confusing variable. Could we change or delete this to be more clear

Reply:
Still find this wording strange

Reply:
@adamhall Any suggestion on improving this?